doc/install_script.sh

```diff
@@ -1072,7 +1072,7 @@ then
     # Install setuptools
     do_setup_py $SETUPTOOLS
 
-    if type -p git &>/dev/null
+    if type -P git &>/dev/null
```

Comment:
hmm, is this a BSD vs Linux option issue?  I see in `bash` both work.  (Not in `zsh` though, which isn't really relevant.)